core/src/main/java/bisq/core/locale/CurrencyUtil.java

```diff
@@ -144,6 +144,7 @@ public static void setBaseCurrencyCode(String baseCurrencyCode) {
         result.add(new CryptoCurrency("GRC", "Gridcoin"));
         if (!baseCurrencyCode.equals("LTC"))
             result.add(new CryptoCurrency("LTC", "Litecoin"));
+        result.add(new CryptoCurrency("LTZ", "LitecoinZ"));
```

Comment:
Do not touch this file

Reply:
I touched this file because I saw ZEC done it. Was it wrong?